### ISSUE TITLE
the arcface loss

### ISSUE THREAD

**Issue by ZHUXUHAN** [lulujianjie/person-reid-tiny-baseline#8]
do you use the arcfaceloss in the project , but I con't find it any where.

**Comment by Jinjicheng** [comment]
how to use arcfaceloss in reid?

**Comment by lulujianjie** [comment]
> do you use the arcfaceloss in the project , but I con't find it any where.

arcface loss is used as a classification layer in model of resnet50. After obtaining logits, you can use softmax loss for training.

**Comment by lulujianjie** [comment]
> how to use arcfaceloss in reid?

You can treat it as linear module, since it has weights for classification. Then try to train the model as a classification task.

**Comment by ZHUXUHAN** [comment]
when I use the arcfacelos，baselr:0.0035 its  gradient update slow, and the acc is always 0, do you know why, please.

**Comment by lulujianjie** [comment]
> when I use the arcfacelos，baselr:0.0035 its gradient update slow, and the acc is always 0, do you know why, please.

Hi~ Arcface loss is a type of metric loss, which aims to learn discrimnative feature embeddings instead of obtain the best acc. Moreover, it can achieve satisfactory acc though it could converge a bit slowly. If you have a dataset with new distribution, you need pay attention to set the hyperparameters, i.e. margin, scale factor, etc.
